src/core/components/hv-route/index.tsx

```diff
@@ -276,7 +276,9 @@ class HvRouteInner extends PureComponent<Types.InnerRouteProps, Types.State> {
    * Evaluate the <doc> element and render the appropriate component
    */
   Route = (): React.ReactElement => {
-    const { isModal } = this.props.route?.params || { isModal: false };
+    const isModal = this.props.route?.params.isModal
+      ? this.props.route.params
```

Comment:
I _think_ this is incorrect
```suggestion
    const isModal = this.props.route?.params.isModal ?? false
```

Reply:
Yes, good catch. Fixed.